src/new-apps/components/AppInProgressDeleteDialog/AppInProgressDeleteDialog.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import Decorator from "@dashboard/storybook/Decorator";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+
+import AppInProgressDeleteDialog, {
+  AppInProgressDeleteDialogProps,
+} from "./AppInProgressDeleteDialog";
+
+const props: AppInProgressDeleteDialogProps = {
+  confirmButtonState: "default",
+  name: "App",
+  onClose: () => undefined,
+  onConfirm: () => undefined,
+  open: true,
+};
+
+storiesOf("Views / Apps / Delete app failed installation", module)
```

Comment:
**suggestion**: Remove this storybook file

We no longer keep `Dialogs` inside storybook - right now we keep only full views there. This file can be removed.

Reply:
DIalogs might be initialized outside of pages, that's why I have left storybook here - on reference comment https://github.com/saleor/saleor-dashboard/pull/2850#discussion_r1054257117 has anything changed since then?